### PR TITLE
Better documentation for callbacks taking arguments

### PR DESCRIPTION
**Pull request by Slava**
This PR makes it possible to use JSDoc's `@callback` directives to declare a specific type of callbacks with specific arguments types.

So far, we still hide the extra arguments in the docs app, but it should be easier to implement with this PR.

(+ it contains some improvements to internal docs)

cc @stubailo

### CODE REVIEW

File: packages/tracker/tracker.js [hunk 1]
```diff
@@ -453,9 +453,13 @@ Tracker.flush = function (_opts) {
 // so that it is stopped if the current computation is invalidated.
 
 /**
+ * @callback TrackerComputationFunction
```
Comment (stubailo) [inline packages/tracker/tracker.js]: Do you think it would be better to namespace it? Like Tracker.ComputationFunction?

**Comment by stubailo** [comment]
LGTM other than the suggested name change, but it might not matter.

The UI for this will be interesting!

**Comment by Slava** [comment]
Corrected, thanks @stubailo, merging it.

**Comment by stubailo** [comment]
+1

**Pull request merged**